doc/cdrwtool.1

```diff
@@ -135,11 +135,29 @@ variable and fixed packet sizes respectively.
 .IP "\fB\-o \fIoffset\fP"
 Set write offset.
 
+.SH BUGS
+Many modern drives refuse on the preparations to format new,
+blanked, or appendable CD-RW media. This causes a message like
+.nf
+  "Command failed: 55 ... - sense ..."
+.fi
+.br
+The remedy is to use a CD-capable burn program for writing a session
+and closing the medium. For example by using any of "cdrecord", 
+"wodim", "cdrskin", or "xorriso -as cdrecord" as content of
```

Comment:
```suggestion
\fBwodim\fP(1), \fBcdrskin\fP(1), or \fBxorriso\fP(1) (via \fB\%xorriso\% -as\% cdrecord\fP) as content of
```
In HTML rendering this should also create symlinks/references to manpages of these programs.